pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go

```diff
@@ -69,11 +221,54 @@ func GetBootstrapSecurityContextConstraints(buildControllerUsername string) []ka
 				// by the admission controller.  Admission will first look for the SupplementalGroupsAnnotation
 				// on the namespace and if it is unable to find that annotation it will attempt
 				// to use the UIDRangeAnnotation.  If neither annotation exists then creation
-				// of the SCC will fail.
+				// of the strategy will fail.
 				Type: kapi.SupplementalGroupsStrategyMustRunAs,
 			},
-			Groups: []string{AuthenticatedGroup},
 		},
+		// SecurityContextConstraintsAnyUID allows no host access and allocates SELinux.
```

Comment:
Only once 1.9 docker lands.  Probably will be optional and user driven.

On Oct 30, 2015, at 3:32 PM, Jordan Liggitt <notifications@github.com>
wrote:

In pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go
<https://github.com/openshift/origin/pull/5543#discussion_r43543249>:

>  		},
> +		// SecurityContextConstraintsAnyUID allows no host access and allocates SELinux.

I thought this was the one we wanted admins to default to... selinux, but
not forcing a uid

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/5543/files#r43543249>.